src/Compilers/CSharp/Portable/Parser/LanguageParser.cs

```diff
@@ -6559,12 +6550,6 @@ private void AccumulateExplicitInterfaceName(ref NameSyntax explicitInterfaceNam
                     separator = this.AddError(separator, ErrorCode.ERR_UnexpectedAliasedName);
                     separator = this.ConvertToMissingWithTrailingTrivia(separator, SyntaxKind.DotToken);
                 }
-                else if (this.CurrentToken.Kind == SyntaxKind.DotDotToken)
```

Comment:
lots of special casing that wanted to treat `..` as `. missing .` just goes away.